File: src_erl/timer.py
import sys
import time

def timer():
    while True:
        line = sys.stdin.readline()
        if not line:
            break
        if line.rstrip() != "time":
            result = "error"
        else:
            result = str(time.time())
        sys.stdout.write("%s\n" % result)
        sys.stdout.flush()

if __name__ == "__main__":
    timer()

File: src_erl/timer.erl
-module(timer).
-export([get_timer/0]).


get_timer() ->
    Port = open_port({spawn, "python -u timer.py"}, [{line, 40}]),
    fun() ->
        port_command(Port, "time\n"),
        receive
            {Port, {data, {eol, Time}}} ->
                {ok, Time}
        after
            1000 ->
                {error, timeout}
        end
    end.